bioscrape/inference_setup.py

```diff
@@ -426,7 +425,8 @@ def run_emcee(self, **kwargs):
         skip_initial_state_check = kwargs.get('skip_initial_state_check', False)
         progress = kwargs.get('progess', True)
         threads = kwargs.get('threads', 1)
-        fname = kwargs.get('results_filename', 'mcmc_results.csv')
```

Comment:
Can we leave this here for backwards compatibility, but maybe put a "Deprecation warning?"

Reply:
Done!